railseventstore.org/source/docs/v2/mapping_serialization.html.md.erb

```diff
@@ -75,6 +77,8 @@ Rails.application.configure do
 end
 ```
 
+Using the `RubyEventStore::NULL` serializer will prevent the event store from serializing the event data and metadata. This is necessary because the Postgres database will handle the serialization.
```

Comment:
To be more precise, ActiveRecord will handle the serialization before putting the data into the database. And will do otherwise when reading. Database itself expect data to be json already.